benchmarks/operator_benchmark/pt/quantization_test.py

```diff
@@ -55,6 +55,42 @@ def forward(self):
                           QuantizePerTensorBenchmark)
 
 
+class QuantizePerChannelBenchmark(op_bench.TorchBenchmarkBase):
+    r"""Benchmarks both quantization and dequantization."""
+    def init(self, C, M, N, dtype, mode):
+        assert(mode in ('Q', 'D'))
+        self.input = torch.rand(C, M, N)
+        self.dtype = dtype
+
+        self.op = torch.quantize_per_channel
+
+        scales = torch.tensor([1.0] * C)
+        zero_points = torch.tensor([0] * C)
+        dtype = dtype
+
+        self.kwargs = {
+            'scales': scales,
+            'zero_points': zero_points,
+            'dtype': dtype,
+            'axis': 0
```

Comment:
Suggest adding one non-zero axis argument as the implementation could be slower when the axis is not zero